src/lib/dialog/dialog-config.ts

```diff
@@ -81,6 +81,11 @@ export class MatDialogConfig<D = any> {
   /** ID of the element that describes the dialog.  */
   ariaDescribedBy?: string | null = null;
 
+  /** Aria label to assign to the dialog element */
```

Comment:
A thought occurred to me (for a follow-up PR if you agree): to make the focus go to the root dialog element, we can just expose a `tabIndex` property on the dialog config. If it's `0`, then it's the first focusable element. Thoughts?

Reply:
I don't think that would work since the interactivity checker starts looking from the children. Also we always focus the dialog anyway in order to prevent the user from opening multiple dialogs while animating.